src/api/special-attributes.md

```diff
@@ -12,7 +12,7 @@
 
   ```html
   <ul>
-    <li v-for="item in items" :key="item.id">...</li>
+    <li v-for="(item , index) in items" :key="index">item</li>
```

Comment:
Using the `index` as a `key` is not really helping much, so `item.id` is a better example.

See also https://rimdev.io/the-v-for-key/ or https://stackoverflow.com/questions/44531510/why-not-always-use-the-index-as-the-key-in-a-vue-js-for-loop.